_clients/k8s-operator.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: default
+title: OpenSearch Kubernetes Operator
+nav_order: 210
+---
+
+The OpenSearch Kubernetes Operator is an open-source kubernetes operator that helps automate the deployment and provisioning of OpenSearch and OpenSearch Dashboards in a containerized environment. The operator can manage multiple OpenSearch clusters that can be scaled up and down depending on your needs. 
+
+
+## Installation 
+
+There are two ways to get started with the operator:
+
+- [Use a Helm chart](#use-a-helm-chartuse-a-helm).
+- [Use a local installation](#use-a-local-installation).
+
+### Use a Helm chart
+
+If you use Helm to manage your Kubernetes cluster, you can use the OpenSearch Kubernetes Operator's Cloud Native Computing Foundation (CNCF) project stored in Artifact Hub, a web-based application for finding, installing, and publishing CNCF packages. 
+
+To begin, log in to your Kubernetes cluster and add the Helm repository (repo) from [Artifact Hub](https://opster.github.io/opensearch-Kubernetes-operator/). 
+
+```
+helm repo add opensearch-operator https://opster.github.io/opensearch-k8s-operator/
+```
+
+Make sure that the repo is included in your Kubernetes cluster. 
+
+```
+helm repo list | grep opensearch
+```
+
+Both the `opensearch` and `opensearch-operator` repos appear in the list of repos.
+
+
+Install the manager that operates all of the OpenSearch Kubernetes Operator's actions. 
+
+```
+helm install opensearch-operator opensearch-operator/opensearch-operator
+```
+
+After the installation completes, the operator returns information on the deployment with `STATUS: deployed`. Then you can configure and start your [OpenSearch cluster](#deploy-a-new-opensearch-cluster).
+
+### Use a local installation
```

Comment:
why can't you use the helm chart just because you're running a local minikube cluster? i think these are separate topics:
* how you run your k8s cluster (real cluster vs. minikube)
* how you deploy the operator (helm vs. self-built)